dora/tests/src/test/java/alluxio/client/rest/TestCase.java

```diff
@@ -76,7 +76,7 @@ public TestCase(String hostname, int port, String baseUri, String endpoint,
     mParameters = parameters;
     mMethod = method;
     mOptions = options;
-    mConnection =  execute();
+    mConnection = execute();
```

Comment:
could you extract execute from the constructor method? We can prepare this connection in the constructor method, and then use the `run()` or `execute` function to actually trigger sending the request.

Reply:
I plan to extract `execute` from the constructor method.